### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'README.md'

### ISSUE THREAD

**Issue by cuihairu** [MaxMaxoff/GracefulKiller#4]
```bash
pip3 install GracefulKiller==0.4
```

error:

```
Collecting GracefulKiller==0.4
  Downloading GracefulKiller-0.4.tar.gz (2.1 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-res09clc/GracefulKiller/setup.py'"'"'; __file__='"'"'/tmp/pip-install-res09clc/GracefulKiller/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-res09clc/GracefulKiller/pip-egg-info
         cwd: /tmp/pip-install-res09clc/GracefulKiller/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-res09clc/GracefulKiller/setup.py", line 5, in <module>
        with open("README.md", "r", encoding="utf-8") as fh:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by MaxMaxoff** [comment]
yep, thank you, the problem that package is not contains README.md. Will fix it soon.

**Comment by MaxMaxoff** [comment]
fixed, version 0.4.6 added
